src/test/java/it/gov/pagopa/payment/test/fakers/SyncTrxStatusFaker.java

```diff
@@ -0,0 +1,13 @@
+package it.gov.pagopa.payment.test.fakers;
+
+import it.gov.pagopa.payment.dto.mapper.TransactionInProgress2SyncTrxStatusMapper;
+import it.gov.pagopa.payment.dto.qrcode.SyncTrxStatusDTO;
+import it.gov.pagopa.payment.enums.SyncTrxStatus;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+
+public class SyncTrxStatusFaker {
+    public static SyncTrxStatusDTO mockInstance(Integer bias, SyncTrxStatus status){
+    TransactionInProgress trx= TransactionInProgressFaker.mockInstance(bias, status);
+        return new TransactionInProgress2SyncTrxStatusMapper().transactionInProgressMapper(trx);
```

Comment:
store the mapper as an instance variable instead of creating it each time

When an object represent a stateless re-usable instance, we could think to save some memory and process effort creating it just once